org.sugarlabs.TypingTurtle.json

```diff
@@ -0,0 +1,268 @@
+                    "url": "http://download.sugarlabs.org/sources/sucrose/glucose/sugar-toolkit-gtk3/sugar-toolkit-gtk3-0.116.tar.xz",
+                    "sha256": "745a360577efceb10f57ca4cbad5f9ec61f7ed6cc381eeb72422bbd7455b770b"
+                }
+            ]
+        },
+        {
+            "name": "sugarapp",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links ${PWD} --install-option=\"--prefix=${FLATPAK_DEST}\" $PWD"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/tchx84/sugarapp.git",
+                    "tag": "v1.10",
+                    "commit": "54b0efdc0bbef60d48685aa2a1559a3fb8f2ddf5"
+                }
+            ]
+        },
+	{
+	    "name":"myspell",
+	    "buildsystem": "simple",
+	    "build-commands" : [
+            "mkdir /app/share/myspell",
+            "cp *.dic /app/share/myspell/."
+	    ],
+	    "sources" : [
+            {
+                "type": "file",
+                "url" : "https://cgit.freedesktop.org/libreoffice/dictionaries/plain/en/en_AU.dic",
+                "sha256": "f8094a6bd24b600518704062e6fd96682cdcb3220d74eff3dd04b99b2ebbb304"
+            },
+            {
+                "type": "file",
+                "url" : "https://cgit.freedesktop.org/libreoffice/dictionaries/plain/en/en_GB.dic",
+                "sha256" : "acc71ce2fd333b0e733923d7426a5ab2edad3745ee419c38f36b7b9816442b08"
+            },
+            {
+                "type": "file",
+                "url" : "https://cgit.freedesktop.org/libreoffice/dictionaries/plain/en/en_US.dic",
+                "sha256" : "e2aec1aff3e2b840c59e2a4f49f7eb9388b6078c681d960153fc72ce183086cc"
+                
+            },
+            {
+                "type": "file",
+                "url" : "https://cgit.freedesktop.org/libreoffice/dictionaries/plain/en/en_ZA.dic",
+                "sha256" : "5928244f918bfcbea2794fac365c0f29de102bd05494115da8301d48ba660571"
+            },
+            {
+                "type": "file",
+                "url" : "https://cgit.freedesktop.org/libreoffice/dictionaries/plain/en/en_CA.dic",
+                "sha256" : "55677b9facf26a8b7b9d0c1099fdf7105c7ed54f8ca9fb85d285c484a996cef0"
+            }
+	    ]
+    },
+	{
+            "name": "typingturtle",
+            "buildsystem" : "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=${FLATPAK_DEST} --skip-install-desktop-file --skip-install-mime"
+            ],
+            "sources" : [
+                {
+                    "type": "git",
+                    "url": "https://github.com/sugarlabs/typing-turtle-activity",
+                    "commit": "62baf1e4989ae8932d67b67e97dca451b4f45bd1"
+                },
+                {
+                    "type": "patch",
+                    "path": "port_flatpak.patch"
```

Comment:
can you rename this to `typingturtle-port.patch` ? so is easier to organize these patches.

Reply:
Fixed on https://github.com/tchx84/flathub/pull/4/commits/168ab52102832246dc420a1f529bf8f4b0df152d